### PR TITLE
B 26 3 disable thymeleaf logs at startup

### PR DESCRIPTION
**Pull request by nomarchia**

### CODE REVIEW

**Comment by Brutt** [comment]
Why did you change pom.xml?

**Comment by nomarchia** [comment]
> Why did you change pom.xml?

I accidentally pushed new changes in pom.xml from the second branch to this branch. I can remove it from here if needed. But anyway same changes would be added after second branch merged into develop,

**Comment by Miari** [comment]
I understund, that jetty part is from other PR, but as it is here I would like to ask why version 9.4.26 is used? Version 9.4.27 is already available

**Comment by Brutt** [comment]
> > Why did you change pom.xml?
> 
> I accidentally pushed new changes in pom.xml from the second branch to this branch. I can remove it from here if needed. But anyway same changes would be added after second branch merged into develop,

Push changes related to branch goal only.

**Review by ValentynaBondarenko — DISMISSED** [review]

File: src/main/resources/logback.xml [hunk 1]
```diff
@@ -27,4 +27,7 @@
     </root>
 
    <!-- TODO: Add console appender    -->
+
+    <!-- Disable ThymeleafEngine logs -->
+   <logger name="org.thymeleaf.TemplateEngine" level="OFF"/>
```
Comment (Rooman) [inline src/main/resources/logback.xml]: Зачем выключать полностью логи тимлифа?

**Review by Rooman — CHANGES_REQUESTED** [review]

**Review by Yuliia-Sh — DISMISSED** [review]

File: src/main/resources/logback.xml (hunk 1 above)
Comment (nomarchia) [reply src/main/resources/logback.xml]: Исправлено на уровень INFO

**Comment by nomarchia** [comment]
> Push changes related to branch goal only.

Sure, of course. That's was my mistake. But since those jetty pom.xml changes already merged to develop branch, do we need to remove it from current branch now? Anyway, it's the same as in the develop branch.

**Comment by nomarchia** [comment]
> I understund, that jetty part is from other PR, but as it is here I would like to ask why version 9.4.26 is used? Version 9.4.27 is already available

By some reason 9.4.27 wasn't available in my Idea, Maven was not finding it. That's why I tried to use 9.2.46

**Review by OleksandrGubariev — DISMISSED** [review]

**Review by Yuliia-Sh — DISMISSED** [review]

**Review by shaguahill — APPROVED** [review]

**Review by OleksandrGubariev — APPROVED** [review]

**Review by Rooman — APPROVED** [review]

**Pull request merged**